examples/flax/language-modeling/run_clm_flax.py

```diff
@@ -558,8 +582,11 @@ def eval_step(params, batch):
     logger.info(f"  Num examples = {len(train_dataset)}")
     logger.info(f"  Num Epochs = {num_epochs}")
     logger.info(f"  Instantaneous batch size per device = {training_args.per_device_train_batch_size}")
-    logger.info(f"  Total train batch size (w. parallel & distributed) = {train_batch_size}")
-    logger.info(f"  Total optimization steps = {total_train_steps}")
+    logger.info(
+        f"  Total train batch size (w. parallel & distributed) = {train_batch_size * training_args.gradient_accumulation_steps}"
+    )
+    logger.info(f"  Total global steps = {total_steps}")
```

Comment:
The number of `total_steps` is the number of optimization steps. Our users are already confused so we need to be consistent with our convention across platforms.